### ISSUE TITLE
Not able to start backend with spaces in script path

### ISSUE THREAD

**Issue by MaggieWang0** [microsoft/chat-copilot#378]
**Describe the bug**
Not able to start the Start-Backend.ps1 when I put the repo in a path with spaces.

**To Reproduce**
1. Put the repo in a path with spaces
2. run '.\Start.ps1' as Readme
3. error: the backend won't be able to start, and the window exist due to no '-NoExit'

**Expected behavior**
Start backend

**Screenshots**
<img width="359" alt="image" src="https://github.com/microsoft/chat-copilot/assets/28853205/56367a65-9958-4a1d-8cfa-947fde3c9cb4">

**Platform**
 - OS: Windows
 - IDE: Powershell
 - Language: ps1
 - Source: main branch of repository

**Additional context**

**Comment by samleegithub** [comment]
I ran into the same problem. Here is a fix to line 10 in Start.ps1:

```
Start-Process pwsh -ArgumentList "-noexit `"$BackendScript`""
```

**Comment by dehoward** [comment]
fixed by #385
